src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -10,4 +10,14 @@ export default interface ClientMetricReport {
    * Gets raw client media metrics
    */
   getObservableMetrics(): { [id: string]: number };
+
+  /**
+   * Gets video upstream client media metrics
+   */
+  getObservableVideoUpstreamMetrics(): { [id: string]: { [id: string]: {} } };
```

Comment:
I think both of these can be collapsed into one getObservableVideoStreamMetrics.